core/sql/cli/Cli.cpp

```diff
@@ -9684,15 +9640,17 @@ Lng32 SQLCLI_LOBcliInterface
 	cliRC = cliInterface->executeImmediate(query);
 
 	currContext.resetSqlParserFlags(0x1);
-
+	
 	if (cliRC < 0)
 	  {
 	    cliInterface->retrieveSQLDiagnostics(myDiags);
-	    
+	    if (myDiags->containsError(-CAT_OBJECT_DOES_NOT_EXIST_IN_TRAFODION))
```

Comment:
We will ignore only this particular  error. If it's any other error, we go to "error_return". That will return an error to the calling layer.